lib/activeadmin_addons/addons/tag_builder.rb

```diff
@@ -1,13 +1,12 @@
 module ActiveAdminAddons
   class TagBuilder < CustomBuilder
     def render
-      @enum_attr = if enumerize_attr?
-                     :enumerize
-                   elsif rails_enum_attr?
+      @enum_attr = if rails_enum_attr?
                      :enum
                    end
 
-      raise "you need to pass an enumerize or enum attribute" unless @enum_attr
+      raise "you need to pass an enum attribute" unless @enum_attr
```

Comment:
with the change in `display_data`, `@enum_attr` is no longer used. We could chage this to just:
```ruby
raise "you need to pass an enum attribute" unless rails_enum_attr?
```